developers/wcs/embeddings/administration.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Administration
+sidebar_position: 3
+image: og/wcs/user_guides.jpg
+---
+
+import Link from '@docusaurus/Link';
+
+Weaviate Embeddings is enabled by default at the organization level and is available to all Weaviate Cloud users. 
+
+## Authentication
+
+In order to use Weaviate Embeddings you only need a [connection to your Weaviate Cloud cluster](/developers/wcs/manage-clusters/connect). 
+No additional authentication is specifically needed, and the Weaviate Embeddings service is enabled by default for all clusters. You can use the service no matter if you connect with a [client library](/developers/weaviate/client-libraries) or, for example, via [OIDC](/developers/weaviate/configuration/authentication#oidc).
+
+## Disable Weaviate Embeddings
+
+Weaviate Embeddings is **enabled by default** at the organization level. To disable the Weaviate Embeddings service for your whole organization, follow these steps:
+
+import DisableWeaviateEmbeddings from '/developers/wcs/img/weaviate-cloud-disable-embeddings.png';
+
+<div class="row">
+  <div class="col col--4">
+    <ol>
+      <li>
+        Open the <Link to="https://console.weaviate.cloud/">Weaviate Cloud console</Link>.
+      </li>
+      <li>
+       Click on <code>Weaviate Embeddings</code> in the left sidebar (<span class="callout">1</span>). 
+      </li>
+      <li>
+       Use the toggle button <code>Enabled</code> to either disable or enable the service. (<span class="callout">2</span>). 
+      </li>
+    </ol>
+  </div>
+  <div class="col col--8">
+    <div class="card">
+      <div class="card__image">
+        <img src={DisableWeaviateEmbeddings} alt="Disable Weaviate Embeddings globally" />
+      </div>
+      <div class="card__body">Disable Weaviate Embeddings globally.</div>
+    </div>
+  </div>
+</div>
+
+<!-- TODO[g-despot] Update screenshot if necessary -->
+
+## Pricing and billing
+
+<!-- TODO[g-despot] Update link -->
+If you would like to learn about the pricing model, you can visit the Weaviate Embeddings [pricing page](/). 
+The pricing works on a per-token basis. This means that you will only be billed for the tokens that are successfully consumed. 
+In other words, only requests that received valid responses from the API are considered.
+
+More info about billing in Weaviate Cloud can be found on [this page](/developers/wcs/platform/billing).
+
+## Rate limits
+
+<!-- TODO[g-despot] Don't hardcode these values here if possible -->
+Weaviate Embeddings only imposes rate limits on requests for Sandbox clusters. 
+The rate limit for Sandbox clusters is `2000` requests in total. 
```

Comment:
I forget - is this per lifetime of the cluster or per day? Either way, I wonder if the "in total" might be somewhat ambiguous.

Reply:
I understood per lifetime of the cluster but asked Charles to confirm. I would replace it with `The rate limit for Sandbox clusters is `2000` requests per cluster.`

Reply:
I think we should explicitly add `for the lifetime of the sandbox` or something like that. Many `rate` limits are per hour/day or so. I'm not sure this is a `rate` either, as it isn't really about throughput.